path-finding.asciidoc

```diff
@@ -12,3 +12,31 @@ Relevant questions to answer:
   * What information can be sent to intermediate and the final node aside from the critical routing data?
   * What are multi-hop locks? What addition privacy and security guarantees to they offer?
   * How can the flexible onion space be used to enabled packet switching in the network?
+
+
+=== What is "Source-Based" routing and why does the Lightning Network use it?
+
+Source-based routing is a method of path-finding where the sender (i.e. the source) plans the route from itself, through the intermediary nodes, and finally to the destination.
+Once a route has been selected, the sender sends the payment to the first intermediary node, who sends it to the second intermediary node and so on, until it reaches the destination.
```

Comment:
nit: once a PATH has been selected...

Reply:
The nit has been picked. See below. Where can I find a reference for the difference between a path and a route? 

https://github.com/lnbook/lnbook/pull/309/commits/33a821d5477dc9303dbd2e60a94016959087b267

Reply:
in mastering the lightning network (: it is still an ongoing battle between terminology but at least we have agreed to make this distinction. and books in general are a good place to set terminology